### ISSUE TITLE
Error opening file on windows, When opening file from File Explorer with directory that containing non ASCII characters.

### ISSUE THREAD

**Issue by aviv1620** [xournalpp/xournalpp#2534]
**Affects versions :**
 - OS: Windows 10
 - libgtk 3.24.33
 - Version of Xournal++:1.0.19
 - Installation method: official release

**Describe the bug**
When i open file from File Explorer with non ASCII characters on directory , i get the message
"Error opening file "c:\?????\test.xopp"
Could not open dile "c:\?????\test.xopp""
the actual directory is  C:\עברית\test.xopp

**To Reproduce**
1. make folder with non ASCII characters.
2. open xournalpp ans save some file in this location,
3. open xopp file from windows file explorer

**Expected behavior**
xournalpp open the file Successfully.

**Screenshots of Problem**
[link](https://ibb.co/c8WLy0c)

**Additional context**
test on Hebrew characters .

**Comment by rolandlo** [comment]
Could you please try out version 1.1.0+dev and see whether the problem is solved there? See [nightly release](https://github.com/xournalpp/xournalpp/releases/tag/nightly)

**Comment by aviv1620** [comment]
yes. I install version 1.1.0+dev now. and is **not solve the problem** .
now I not get the error message but is open blank document instead of my notes.

**Comment by MichelReims** [comment]
Sometimes, I have the same problem. I can't reproduce at every time

**Comment by Febbe** [comment]
We actually won't fix that issue anymore for v.1.0.x, but I will look into it for 1.1.0(-dev). 
Thanks for the report, I thought I fixed it already, at least for iso-8859-1.

**Comment by Febbe** [comment]
Can you tell me your local locale?

**Comment by aviv1620** [comment]
English (United States)
